### DISCUSSION TITLE
"Metaschema syntax" page inconsistency on `Field` definition `@required` parameter

### DISCUSSION THREAD

**Discussion by nikitawootten-nist** [usnistgov/metaschema#309] [Q&A] [answered]
Hi all, I'm looking over the "Metaschema Syntax" page under the heading "Metaschema modeling" (https://pages.nist.gov/metaschema/specification/syntax/#metaschema-modeling). **Is the following excerpt a typo**:

https://github.com/usnistgov/metaschema/blob/529ff85fd10985c850e420de2a0f33fd84fc3663/website/content/specification/syntax/_index.md?plain=1#L680

AFAIK `@required` **only applies to flags**? "Requirement" is to be handled in fields and assemblies using `@{min,max}-occurs`? The capability to represent `@required` is *not* present in metaschema-java: https://pages.nist.gov/metaschema-java/apidocs/gov/nist/secauto/metaschema/model/common/IFieldInstance.html

Feel free to convert this into an issue and I can volunteer to open a PR if this is indeed a typo :)

**Comment by wendellpiez** [comment, marked as answer]
Yes, this is a typo. @min-occurs would render the value either superfluous, or contradictory.
